### ISSUE TITLE
ImportError: cannot import name MPVaspInputSet

### ISSUE THREAD

**Issue by harklee** [materialsproject/MPWorks#16]
After updates to  pymatgen.io.vasp.sets, lpad commands to run or modify workflows
seem to encounter the following errors. Also some of the input parameter yaml files
in pymatgen.io.vasp have disappeared in recent releases.

File "/global/u1/h/harklee/hark_inter/codes/MPWorks/mpworks/firetasks/vasp_setup_tasks.py", line 8, in <module>
    from pymatgen.io.vasp.sets import MPVaspInputSet, MPStaticVaspInputSet, \
ImportError: cannot import name MPVaspInputSet

**Comment by tschaume** [comment]
Hi Hark,

MPWorks is not compatible with pymatgen 4 (yet). Please downgrade to an
earlier version.

Best,
Patrick

**Comment by harklee** [comment]
Yes, Patrick,

Looks like the whole bunch of input sets are being depreciated and sorted out. I looked for which would be the best head # to revert pymatgen to, and a63bcdf seems to work fine. That's just before all the major changes were made. Thanks.

**Comment by shyuep** [comment]
No, you should not be reverting pymatgen. You should pin MPWorks to an older pymatgen version.

Note that I gave deprecation warning messages for 3-6 months, and stated very clearly that the switch will occur on Jul 1. The MPworks incompatibilities should have been sorted out long before today. I am not going to hold back pymatgen upgrade because of this.

**Comment by xhqu1981** [comment]
I agree. My NMR workflows are implemented using the new style input set.
Now I have great fear to break existing workflows by merging my branch.

**Comment by tschaume** [comment]
yes, Shyue, I'm aware. Thanks.
In Hark's case, he installed pymatgen from MPenv which installs the development
version from GitHub and I just wanted to get him going again as quickly as
quickly as possible.





On Fri, Jul 1, 2016 2:32 PM, Shyue Ping Ong notifications@github.com wrote:
No, you should not be reverting pymatgen. You should pin MPWorks to an older
pymatgen version.

Note that I gave deprecation warning messages for 3-6 months, and stated very
clearly that the switch will occur on Jul 1. The MPworks incompatibilities
should have been sorted out long before today. I am not going to hold back
pymatgen upgrade because of this.
